src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/components/ProfileForm.vue

```diff
@@ -67,7 +67,7 @@ export default defineComponent({
       if (!valid) {
         this.alertStore.setFailureAlert("You must enter all required fields in the valid format.");
       } else {
-        const success = await putProfile({
+        const { error } = await putProfile({
```

Comment:
I think line 83 needs to be changed? I don't see the success variable being declared.

Reply:
Yes, thanks bad merge